miniapps/plasma/cold_plasma_dielectric_solver.cpp

```diff
@@ -582,11 +583,17 @@ CPDSolver::CPDSolver(ParMesh & pmesh, int order, double omega,
    */
    // Bilinear Forms
    a1_ = new ParSesquilinearForm(HCurlFESpace_, conv_);
+   if (pa_)
+   {
+      a1_->real().SetAssemblyLevel(AssemblyLevel::PARTIAL);
+      a1_->imag().SetAssemblyLevel(AssemblyLevel::PARTIAL);
```

Comment:
Can we add a method `ParSesquilinearForm::SetAssemblyLevel` to make this a single line?

Reply:
I recall that we had already added `ParSesquilinearForm::SetAssemblyLevel` on your request in a different PR that's been merged into master, see https://github.com/mfem/mfem/commit/3e6889145ea7fed5eea05fbe0cb8bf5b05bfd660. I'm changing it here to simply call that member function.

Reply:
I thought so too but my memory is unreliable.